CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,53 @@
+# Meetup Code of Conduct
+
+All organizers, attendees, speakers, sponsors and volunteers at Kamloops R
+Meetup events are required to agree to uphold the following code of conduct.
+Kamloops R Meetup event organizers and/or Kamloops Innovation staff will enforce
```

Comment:
Have we confirmed with Beth or Kimberley about having KIC help in enforcing the Code of Conduct?